src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPremiumPrerollMOATTracking.java

```diff
@@ -14,33 +14,33 @@
   private static final String IGNORE_SAMPLING = "ignored_samplers=moatTrackingForFeaturedVideo";
   private static final String QUERY_PARAM_CONTROL_GROUP = "AbTest.MOBILE_FEATURED_VIDEO_AUTOPLAY=CONTROL";
   private static final String MOAT_VIDEO_TRACKING_URL = "https://z.moatads.com/wikiaimajsint377461931603/moatvideo.js";
-  private static final String FEATURED_VIDEO_WITH_MOAT_PATH_FORMAT = "%s?" + TURN_ON_MOAT + "&" + IGNORE_SAMPLING + "&" + QUERY_PARAM_CONTROL_GROUP;
+  private static final String FEATURED_VIDEO_WITH_MOAT_PATH_FORMAT = TURN_ON_MOAT + "&" + IGNORE_SAMPLING + "&" + QUERY_PARAM_CONTROL_GROUP;
```

Comment:
It's not a format anymore (no `%s` or other placeholders) but `FEATURED_VIDEO_WITH_MOAT_PATH_PARAMETER`, I guess :)